--- ubiquity/ubiquity/frontend/gtk_ui.py	2023-06-05 18:00:53.000000000 -0300
+++ ubiquity/ubiquity/frontend/gtk_ui.py.new	2024-07-14 21:32:07.061510709 -0300
@@ -969,8 +969,6 @@
         return False
 
     def start_slideshow(self):
-        # WebKit2 spawns a process which we don't want to run as root
-        misc.drop_privileges_save()
         self.progress_mode.set_current_page(
             self.progress_pages['progress_bar'])
         telemetry.get().add_stage('user_done')
@@ -979,6 +977,8 @@
             self.page_mode.hide()
             return
 
+        # WebKit2 spawns a process which we don't want to run as root
+        misc.drop_privileges_save()
         self.page_section.hide()
 
         slideshow_locale = self.slideshow_get_available_locale(
